app/src/main/java/net/squanchy/service/repository/firebase/FirebaseEventRepository.kt

```diff
@@ -0,0 +1,67 @@
+package net.squanchy.service.repository.firebase
+
+import io.reactivex.Completable
+import io.reactivex.Observable
+import io.reactivex.functions.Function3
+import io.reactivex.schedulers.Schedulers
+import net.squanchy.schedule.domain.view.Event
+import net.squanchy.service.firebase.FirebaseDbService
+import net.squanchy.service.firebase.model.FirebaseFavorites
+import net.squanchy.service.firestore.FirestoreDbService
+import net.squanchy.service.firestore.model.conferenceinfo.FirestoreVenue
+import net.squanchy.service.firestore.model.schedule.FirestoreEvent
+import net.squanchy.service.firestore.toEvent
+import net.squanchy.service.repository.EventRepository
+import net.squanchy.support.lang.Checksum
+import org.joda.time.DateTimeZone
+
+class FirebaseEventRepository(
+    private val dbService: FirebaseDbService,
```

Comment:
:x: using the favourites from RTDB is most likely _not_ going to be working. We should leave them empty for now! There's a TODO for them in #367.

Reply:
Fixed